packages/next/next-server/server/next-server.ts

```diff
@@ -315,6 +323,76 @@ export default class Server {
       },
     ]
 
+    let rewrites: {
+      source: string
+      destination: string
+    }[] = []
+    let redirects: {
+      source: string
+      destination: string
+      statusCode?: number
+    }[] = []
+
+    if (this.dev) {
```

Comment:
This should be handled in `next-dev-server` instead of branched here, correct?

Reply:
Yup, was there just for testing purposes. Updated